commit 7c6070fee5ab0b77d6e47e4cd17caf2b2578f8a5
Author: John Doe <jdoe@Johns-Mac.local>
Date:   Wed Feb 8 19:10:11 2017 -0500

    do not reference gtk-doc.make from docs/Makefile.am or configure.ac

diff --git a/configure.ac b/configure.ac
index fc6cc6b..2a6cb25 100644
--- a/configure.ac
+++ b/configure.ac
@@ -490,7 +490,7 @@ if $have_cairo ; then : ; else
    fi
 fi
 
-GTK_DOC_CHECK([1.15], [--flavour no-tmpl])
+#GTK_DOC_CHECK([1.15], [--flavour no-tmpl])
 
 AC_ARG_ENABLE(doc-cross-references,
               AC_HELP_STRING([--disable-doc-cross-references],
diff --git a/docs/Makefile.am b/docs/Makefile.am
index f5f1317..7e72401 100644
--- a/docs/Makefile.am
+++ b/docs/Makefile.am
@@ -98,7 +98,8 @@ FIXXREF_OPTIONS=--extra-dir=$(GLIB_PREFIX)/share/gtk-doc/html/glib \
 	--extra-dir=$(CAIRO_PREFIX)/share/gtk-doc/html/cairo
 endif
 
-include $(top_srcdir)/gtk-doc.make
+#include $(top_srcdir)/gtk-doc.make
+EXTRA_DIST =
 
 # This line really belongs in gtk-doc.mk
 $(REPORT_FILES): sgml-build.stamp
@@ -139,9 +140,9 @@ endif
 
 .PHONY: dist-local-check-no-cross-references
 
-if ENABLE_GTK_DOC
-TESTS = check.docs
-TESTS_ENVIRONMENT = srcdir="$(srcdir)" top_srcdir="$(top_srcdir)" MAKE="$(MAKE) $(AM_MAKEFLAGS)" DOC_MODULE="$(DOC_MODULE)" REPORT_FILES="$(REPORT_FILES)"
-endif
+#if ENABLE_GTK_DOC
+#TESTS = check.docs
+#TESTS_ENVIRONMENT = srcdir="$(srcdir)" top_srcdir="$(top_srcdir)" MAKE="$(MAKE) $(AM_MAKEFLAGS)" DOC_MODULE="$(DOC_MODULE)" REPORT_FILES="$(REPORT_FILES)"
+#endif
 
 -include $(top_srcdir)/git.mk
